coalib/parsing/DefaultArgParser.py

```diff
@@ -243,14 +243,27 @@ def default_arg_parser(formatter_class=None):
         '-r', '--relpath', nargs='?', const=True,
         help='return relative paths for files (must be called with --json)')
 
-    outputs_group.add_argument(
+    exclusive_group = arg_parser.add_mutually_exclusive_group()
+
+    exclusive_group.add_argument(
         '--debug-bears', nargs='?', const=True,
         help='Enable bear debugging with pdb, that can help to identify and'
         ' correct errors in bear code. Steps into bear code as soon as being'
         ' executed. To specify which bears to debug, supply bear names as'
         ' additional arguments. If used without arguments, all bears specified'
         ' with --bears will be debugged (even implicit dependency bears).')
 
+    exclusive_group.add_argument(
+        '--profile', nargs='?', const=True,
+        help='Enable bear profiling with cProfile, that can help to create'
+        ' performant bears by profiling Bear code. To specify where to dump the'
+        ' profiled files, supply the directory path. If specified directory'
+        ' does not exist it will be created. If the specified path points to an'
+        ' already existing file a ``FileExitsError`` is thrown . All bears'
```

Comment:
If user provide a `existing file` path, doesn't `FileExitsError` have to raise?
In verbose mode it will traceback information like this 
```
Traceback information is provided below:

Traceback (most recent call last):
  File "/home/vaibhav/4/lib/python3.6/site-packages/coalib/bears/Bear.py", line 436, in execute
    result = self.run_bear_from_section(args, kwargs)
  File "/home/vaibhav/4/lib/python3.6/site-packages/coalib/bears/Bear.py", line 417, in run_bear_from_section
    return self.profile_run(*args, **kwargs)
  File "/home/vaibhav/4/lib/python3.6/site-packages/coalib/bears/Bear.py", line 398, in profile_run
    self._dump_bear_profile_data(profiler)
  File "/home/vaibhav/4/lib/python3.6/site-packages/coalib/bears/Bear.py", line 381, in _dump_bear_profile_data
    os.makedirs(self.profile)
  File "/home/vaibhav/4/lib/python3.6/os.py", line 220, in makedirs
    mkdir(name, mode)
FileExistsError: [Errno 17] File exists: 'abc.txt'
```

Reply:
1. `FileExistsError` not `FileExitsError`
2. 'thrown' in the context of `--help` means that the application will *not* catch the exception and will fail with a Python backtrace.  If an exception is handled within the application, it is not relevant within `--help` and you need to revise `--help` to explain what the *application* will do, i.e. how does it handle the `FileExistsError` inside the application.

Reply:
>FileExistsError not FileExitsError

:+1: 

>'thrown' in the context of --help means that the application will not catch the exception and will fail with a Python backtrace

Yeah this is the current scenario, we are agreed on not handling the exception instead failing with traceback.